examples/common/python/utility/signature.py

```diff
@@ -282,22 +282,25 @@ def generate_client_signature(self, input_json_str,
         return input_json_str, SignatureStatus.PASSED
 
 #---------------------------------------------------------------------------------------------
-    def verify_signature(self,response_str,worker):
+    def verify_signature(self,input_json,worker):
```

Comment:
Do we need to pass entire worker object to thsi function ? can't we just pass worker's HashingAlgorithm and SigningAlgorithm ?

Reply:
Ideally verify_signature() it would take only 2 parameters verifying key and signature. It is checking additional thing by verifying the algorithms for hashing and signing. Do we really need that for verifying the signature?

Reply:
It's good to pass hashing and signing to verify_signature() function since it does hashing/signing operations and it uses some default hashing/signing algorithms.